routes/web.php

```diff
@@ -33,66 +34,71 @@
         return view('dashboard');
     })->name('dashboard');
 
-    Route::get('/log/mine', [ActivityLogController::class, 'mine'])
-        ->name('activity_log.mine');
+    Route::get('/activities', [ActivitiesController::class, 'index'])
+        ->name('activities');
 
-    Route::get('/teams', [TeamsController::class, 'index'])
-        ->name('teams');
+    Route::controller(ActivityLogController::class)
+        ->name('log')
+        ->group(function () {
+            Route::get('/log', 'viewLog');
+            Route::get('/log/{activity}', 'logActivity')
+                ->name('.activity');
+        });
 
-    Route::get('/activities/create', [ActivitiesController::class, 'create'])
-        ->name('activities.create');
-    Route::post('/activities/create', [ActivitiesController::class, 'store'])
-        ->name('activities.store');
+    Route::get('/users/{user}/edit', [UsersController::class, 'edit'])
```

Comment:
@drraccoony I'd like to refrain from having the user id in the url for the basic user experience, maybe there should be two routes, one for administration with the user id and another for standard users editing their own profile.